runbotics/runbotics-orchestrator-ui/src/src-app/views/process/ProcessBuildView/Modeler/ActionFormPanel/widgets/ElementAwareAutocompleteWidget.tsx

```diff
@@ -6,110 +6,18 @@ import { WidgetProps } from '@rjsf/core';
-    'environment.services.objectAsBoolean',
-    'environment.services.stringObjectAsBoolean',
-    'environment.services.orObjectsAsBoolean',
-    'environment.services.andObjectsAsBoolean',
-    'environment.services.isEmptyString',
-    'environment.services.isEqualStr',
-    'environment.services.isEqual',
-    'environment.services.isNotEqual',
-    'environment.services.incrementAndCompare',
-    'environment.services.isBlank',
-    'environment.services.push',
-    'environment.services.concat',
-    'environment.services.splitAndPick',
-    'environment.services.parseJson',
-    'environment.services.csvToObject',
-    'environment.services.comaSeparatedToArray',
-    'environment.services.applyConverters',
-    'environment.services.withDefault',
-    'environment.services.getCurrentDate',
-    'environment.services.getPreviousWorkday',
-    'environment.services.checkCurrentTimeBefore',
-    'environment.services.checkCurrentTimeAfter',
-    'environment.services.checkCurrentTimeBetween',
-    'environment.services.getSecondsToNextDayTime',
-    'environment.services.getSecondsToSpecifiedHour',
-    'environment.services.getFirstWorkingDayOfCurrentMonth',
-    'environment.services.getFirstWorkingDayOfFutureMonth',
-    'environment.services.getLastWorkingDayOfCurrentMonth',
-    'environment.services.getLastWorkingDayOfFutureMonth',
-    'environment.services.checkFirstWorkingDayOfCurrentMonth',
-    'environment.services.checkLastWorkingDayOfCurrentMonth',
-    'environment.services.getSecondsToFirstWorkingDayOfMonth',
-    'environment.services.transformArraysToArray',
-    'environment.services.getCurrentShortMonthYear',
-    'environment.services.compareMaps',
-    'environment.services.diffMaps',
-    'environment.services.objFromArray',
-    'environment.services.readFromStorage',
-].map((service) => ({
-    value: `\${${service}}`,
-    label: `\${${service}}`,
-    group: t(
-        'Process.Details.Modeler.Widgets.ElementAwareAutocomplete.Groups.Services'
-    ),
-}));
-
-const UTILS = [
-    'false',
-    'true',
-    'content.output',
-    'environment',
-    'environment.output',
-].map((util) => ({
-    value: `\${${util}}`,
-    label: `\${${util}}`,
-    group: t(
-        'Process.Details.Modeler.Widgets.ElementAwareAutocomplete.Groups.Utils'
-    ),
-}));
-
-const VARIABLES = [
-    'tempFolder',
-    'userEmail',
-].map(variable => ({
-    label: variable,
-    value: variable,
-    group: t(
-        'Process.Details.Modeler.Widgets.ElementAwareAutocomplete.Groups.Variables'
-    ),
-}));
-
-const reduceList = (list: any[]) =>
-    list.reduce((previousValue, currentValue) => {
-        const newPrev = previousValue;
-        newPrev[currentValue.value] = currentValue;
-        return newPrev;
-    }, {});
-
 interface ElementAwareAutocompleteProps extends WidgetProps {
     options: {
         info?: string;
     };
     customErrors?: string[];
+    autocompleteOptions: Options;
```

Comment:
autocomplete is obvious prefix

Reply:
I wanted to differentiate this prop name to be clear what those options are for, it is because there is already one prop named 'options' and it comes as default Form component from rjsf lib.

Reply:
is this default options {} used?

Reply:
if yes then keep autocompleteOptions

Reply:
Yes, it is passed down to another compononent